### PR TITLE
Add namespace to procedure names

### PR DESCRIPTION
**Pull request by Mats-SX**
- Correctly use parentheses for in-query procedure call

### CODE REVIEW

File: grammar/basic-grammar.xml [hunk 1]
```diff
@@ -370,9 +370,16 @@
   </production>
 
   <production name="ProcedureName" rr:inline="true">
+    <non-terminal ref="Namespace" rr:title="procedure namespace"/>
     <non-terminal ref="SymbolicName" rr:title="procedure name"/>
   </production>
 
+  <production name="Namespace" rr:inline="true">
+    <repeat min="1">
```
Comment (thobe) [inline grammar/basic-grammar.xml]: Whether `min` should be `0` or `1` is still being debated is it not?

Comment (Mats-SX) [inline grammar/basic-grammar.xml]: It is.

Comment (Mats-SX) [inline grammar/basic-grammar.xml]: @thobe Would you be more comfortable merging this is the `min` is set to `0`?

**Comment by Mats-SX** [comment]
@thobe Changed it so that namespace can be empty.

**Pull request merged**